### DISCUSSION TITLE
Notification replace

### DISCUSSION THREAD

**Discussion by semi710** [ErikReider/SwayNotificationCenter#234] [Q&A] [answered]
I got the point that we can replace notification using the id
but then I have to do `--print-id` and save that to a variable to replace the notification.

where as in mako or dunst I can replace them using tag
e.g 
```
  notify \
    --hint=string:x-dunst-stack-tag:volume \
    -a "Volume" -u low \
    -h int:value:"$volume" "Volume: ${volume}" \
    -t 1000
```
Now I would like to use something like this as I use notification manager to show my volume percentage 
and I call that script multiple times to increase volume .
[script](http://ix.io/4qNK) for reference

**Comment by ErikReider** [comment, marked as answer]
There's something like this builtin which uses a more "official" notification hint.

Here's a loop that counts up to 100 :)

```sh
seq 100 | while read number; do
  notify-send -h int:value:$number -h string:synchronous:my-progress "Progress" 
  sleep 0.3
done
```

**Reply by semi710** [reply]
Exactly what I needed thanks.
